monitoring/test-redis-metrics.sh

```
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Load environment variables
+if [ -f .env ]; then
+    export $(cat .env | grep -v '^#' | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)